tests/Transcription.IntegrationTests/DeepgramTranscriberTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿using ActualChat.Audio;
+using ActualChat.IO;
+using ActualChat.Transcription.DeepGram;
+using ActualChat.Transcription.Module;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.EnvironmentVariables;
+using Stl.IO;
+using Stl.Testing.Output;
+using Xunit.DependencyInjection.Logging;
+
+namespace ActualChat.Transcription.IntegrationTests;
+
+public class DeepgramTranscriberTest(ITestOutputHelper @out, ILogger<DeepGramTranscriber> log): TestBase(@out)
+{
+    private ILogger<DeepGramTranscriber> Log { get; } = log;
+
+    [Theory(Skip = "Manual")]
+    // [InlineData("file.webm", false)]
+    [InlineData("file.webm", true)]
+    // [InlineData("0002-AK.opuss", true)]
```

Comment:
why are these cases off?

Reply:
This is a manual only test - so other options are useful for manual checks